### PR TITLE
Aido1 lf1 r3 v3 revision

### PR DESCRIPTION
**Pull request by jzilly**
Changing the "good_angle" and "valid_direction" score by dividing them by the "survival time". This will make them "average over time" scores which should be easier to interpret. Currently, it may appear as if higher is better since a higher value is linked to a longer survival time.

### CODE REVIEW

**Comment by liampaull** [comment]
do we need to change a "higher-is-better" to a "lower-is-better" in the definition of the challenge then @AndreaCensi ?

**Comment by AndreaCensi** [comment]
I don't get it - it should be lower is better.

**Comment by AndreaCensi** [comment]
@liampaull in general these are hacks until we get proper lane support in the simulator

File: scorer/read_scores.py [hunk 1]
```diff
@@ -57,6 +57,14 @@ def compute_rules_violation(log_file, cie):
         if 'lane_position' in Simulator:
             lane_position = Simulator['lane_position']
             angle_rad = lane_position['angle_rad']
+
+            # Added logic to ensure angle w.r.t. lane
+            # is irrespective of direction of travel
```
[the remark this answers is not in the document]
Comment (jzilly) [reply scorer/read_scores.py]: Well it is currently restricting the direction of travel per road. If I understand correctly, the simulator assumes that there is a "correct" lane and the direction of that lane is where you should go. In reality, you could very well follow the other lane in the opposite direction. Hence the change.

[the remark this answers is not in the document]
Comment (jzilly) [reply scorer/read_scores.py]: also thinking it should be "the lower-the-better" for "good_angle" and "valid_direction"

**Comment by liampaull** [comment]
@AndreaCensi is there an issue for that assigned to someone ?

**Comment by AndreaCensi** [comment]
closing this - fixed otherwise

**Pull request closed without merging**